Formula/livekit.rb

```diff
@@ -0,0 +1,25 @@
+class Livekit < Formula
+  desc "Scalable, high-performance WebRTC SFU. Written in pure Go"
```

Comment:
```suggestion
  desc "Scalable, high-performance WebRTC SFU (Selective Forwarding Unit)"
```
Being written in Go is an implementation detail.

Reply:
updated, PTAL